index.js

```diff
@@ -35,7 +35,7 @@ module.exports = function (file, isBrowser = false, update, options) {
   var workerRunning = true;
   return new Promise(function (resolve, reject) {
     var readBlock = readBlockFactory();
-    mp4boxFile = MP4Box.createFile(false);
+    mp4boxFile = MP4Box.createFile();
```

Comment:
Why is the **false** removed here? I don't remember why it was there in the fist place, but just to be sure that we are not breaking anything. Thank you